data/campaigns/Descent_Into_Darkness/scenarios/01_Saving_Parthyn.cfg

```diff
@@ -393,6 +394,7 @@
 
         {CLEAR_VARIABLE malin}
 
+        # FIXME: allow_undo like this causes corrupt replays.
```

Comment:
Removing the [allow_undo] on this feels right to me, but it doesn't have the definite reason that the other one does.

@nemaara as this gives a significant amount of XP, I think there should be at least some hint if the player sends a walking corpse to investigate it.

Reply:
All of the [allow-undo]s can just be removed I think.